ocw/lib/gce.py

```diff
@@ -21,9 +21,11 @@ def __new__(cls, namespace):
     def __init__(self, namespace):
         super().__init__(namespace)
 
+        self.__bucket = PCWConfig.get_feature_property('cleanup', 'gce-bucket', namespace)
```

Comment:
we also need to support case when there is pcw.ini where this variable is NOT defined .

Reply:
Fixed.